elide-datastore/elide-datastore-hibernate3/src/main/java/com/yahoo/elide/datastores/hibernate3/HibernateTransaction.java

```diff
@@ -113,6 +114,33 @@ public void commit() {
         return list;
     }
 
+    /**
+     * Internal loadObjects helper - loads pagination objects
+     * @param loadClass The class Type reference
+     * @param criterion The optional Criterion
+     * @param pagination The Pagination object
+     * @param <T> The class type
+     * @return The ScrollableIterator
+     */
+    public <T> Iterable<T> loadObjects(Class<T> loadClass, Criterion criterion, Pagination pagination) {
```

Comment:
true. I will add that functionality as well. The first pass was simple, the next pass will include the /author/id/books?page[number]=2 functionality.